tests/testMixture.py

```diff
@@ -37,18 +37,16 @@
 except ImportError:
     scipy = None
 
-numpy.random.seed(500)
-rng = lsst.afw.math.Random("MT19937", 500)
-
 class MixtureTestCase(lsst.utils.tests.TestCase):
 
     @staticmethod
     def makeRandomMixture(nDim, nComponents, df=float("inf")):
+        numpy.random.seed(500)
```

Comment:
I think it would be better to put this line in a `setUp` method rather than duplicate it in each test.